ui/hooks/useCarouselManagement/useCarouselManagement.ts

```diff
@@ -54,6 +55,7 @@ export const useCarouselManagement = ({
       undismissable: hasZeroBalance,
     };
 
+    defaultSlides.push(SMART_ACCOUNT_UPGRADE_SLIDE);
```

Comment:
Minor, do we need to go in a code fence below?

Reply:
Good point, I made the change